content/blog/2025-12-16-facade-layout-compiler.md

```diff
@@ -0,0 +1,409 @@
++++
+title = "Facade Layout Compiler (# 618)"
```

Comment:
No need for the issue number here. Remember that this blog post is for public consumption; the public doesn't care about GitHub internals.